spec/ttt/board_spec.clj

```diff
@@ -0,0 +1,19 @@
+(ns ttt.board-spec
+  (:require [speclj.core :refer :all]
+            [ttt.board :refer :all]))
+
+(describe "Board"
+          (it "creates a new empty board"
+              (should= [0 1 2 3 4 5 6 7 8] (initial-board)))
+          (it "places a move on the board"
+              (should= ["X" 1 2 3 4 5 6 7 8] (place-marker [0 1 2 3 4 5 6 7 8] 0 "X")))
+          (it "checks if a position is available"
+              (should= false (position_available? ["X" 1 2 3 4 5 6 7 8] 0))
+              (should= true (position_available? ["X" 1 2 3 4 5 6 7 8] 1)))
```

Comment:
It may be helpful to have describe blocks per function and one `it` block for each of these assertions with descriptive test names!